packages/winpanda/extra/src/winpanda/bin/winpanda.py

```diff
@@ -0,0 +1,120 @@
+"""Panda package management for Windows.
+
+CLI entry point definition.
+"""
+from pathlib import Path
+import sys
+
+from docopt import docopt, DocoptExit
+
+sys.path.insert(1, f'{Path(sys.path[0]).parent}')
+
+from common import constants as cm_const
+from common.cli import (
+    CLI_ARGSPEC, CLI_COMMAND, CLI_CMDTARGET, VALID_CLI_CMDTARGETS, CLI_CMDOPT
+)
+from common import storage
+from common import logger
+from common.logger import LOG_LEVEL
+from core import command
+
+
+LOG = logger.get_logger(__name__)
+
+
+class DCOSInstallationManager:
+    """DC/OS installation manager."""
+    def __init__(self):
+        """Constructor."""
+        self.cli_args = self._get_cli_args()
```

Comment:
We don't need to store `cli_args` as an attribute. Just pass it straight to the `_get_command` method.

Reply:
As winpanda is under active development for the moment I'd keep it as it is for providing better visual perception of which entities the app is operating with. And this approach also follows PEP20's rule of 'explicit is better than implicit' and so suport better code readability ... to my mind.